doc/design.md

```diff
@@ -23,54 +23,50 @@ first version (V1) of the FHIR gateway:
 
 <img src="summary.png" width=50% height=50%>
 
-*   There are three main components responsible for access decisions: an
```

Comment:
I tried pulling the main again. There is nothing to update it says. I feel this might be a Github issue particularly for PRs. I tried comparing the `google:main` with `anchitag:smart-patient-scopes` and I don't see the files that are not supposed to be there in this comparision [here](https://github.com/google/fhir-gateway/compare/main...anchita-g:fhir-access-proxy:smart-patient-scopes).

Reply:
I see, yeah that is weird; these changes should not be part of this PR (as they are already in the `main` branch). Anyway, not a big deal, if it is not possible to resolve it, just merge the PR.